oxidation/libparsec/crates/crypto/Cargo.toml

```diff
@@ -41,6 +41,7 @@ serde_bytes = { version = "0.11.9" }
 hex = "0.4"
 thiserror = "1.0.40"
 base32 = "0.4.0"
+sharks = "0.5.0"
```

Comment:
We use a dependencies that isn't updated in `2021` ?

Reply:
Good point

@TimeEngineer did you consider other libs such as https://github.com/geky/gf256? What criteria did you use to make the decision?

Reply:
I took the most popular crate: https://crates.io/keywords/shamir